modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/util/NavigationItem.java

```diff
@@ -81,6 +82,10 @@ public void setHref(PortletURL portletURL, Object... parameters) {
 		setHref(portletURL.toString());
 	}
 
+	public void setIcons(List<IconItem> icons) {
+		put("icons", icons);
+	}
+
```

Comment:
I think a better place for this would be `VerticalNavItem`. `NavigationItem` is more generic, `icons` is not applicable to all variants that extend it.